### ISSUE TITLE
Distortions in Synthesis mode 

### ISSUE THREAD

**Issue by fatihkiralioglu** [Rayhane-mamah/Tacotron-2#295]
Hi,
I have trained a Tacotron-2 model over a custom English database containing 13 hours of speech. 

I have used default configuration. My 945000th step synthesis results are generally very good. However, in some of the synthesis records I have observed some distortions as can be seen in the attachment.

I wonder if there is anyone else getting similar results and I welcome any advice to solve this.

Best Regards.

[wavs_945k_samples.zip](https://github.com/Rayhane-mamah/Tacotron-2/files/2671018/wavs_945k_samples.zip)

**Comment by sid-sundrani** [comment]
Interesting how the ends are only distorted. In your training set, are the ends of your audio samples perfectly clipped? Meaning, does the speaker completely utter the last word in every sample?

**Comment by sid-sundrani** [comment]
I don't think it should matter too much because of the attention mechanism. You should try using the trim_silence preprocessing parameter

**Comment by fatihkiralioglu** [comment]
Thanks Sid, I'm already using trim_slience parameter and i will check if the training audio files are correctly trimmed.

**Comment by fatihkiralioglu** [comment]
I have checked the trimmed versions of the database records and i have observed that for some of the files, trimming process cuts a portion of the first and last phonemes in records. I will try to change the trimming parameters to improve the trimming performance.





[trimmed_samples.zip](https://github.com/Rayhane-mamah/Tacotron-2/files/2701959/trimmed_samples.zip)

**Comment by Coastchb** [comment]
hi, @fatihkiralioglu 
Do the distortions at the end of synthesized wav completely disappear now? 
And at the best parameter setting, are all the leading and tailing silence of training wav trimmed?

**Comment by fatihkiralioglu** [comment]
Hi @Coastchb , 
I did not try a new training with changed trimming parameters, currently i work on some existing github implementations of parallel wavenet and clarinet in order to get a fast synthesis pipeline using wavenet. There are very few work on that side as far as i have observed.
